lib/datasource/galaxy/index.ts

```diff
@@ -30,65 +35,73 @@ export async function getPkgReleases({
     '&name=' +
     projectName;
   const galaxyProjectUrl = baseUrl + userName + '/' + projectName;
+  try {
+    let res: any = await got(galaxyAPIUrl, {
+      hostType: id,
+    });
+    if (!res || !res.body) {
+      logger.warn(
+        { dependency: lookupName },
+        `Received invalid crate data from ${galaxyAPIUrl}`
+      );
+      return null;
+    }
 
-  let res: any = await got(galaxyAPIUrl, {
-    hostType: id,
-  });
-  if (!res || !res.body) {
-    logger.warn(
-      { dependency: lookupName },
-      `Received invalid crate data from ${galaxyAPIUrl}`
-    );
-    return null;
-  }
-
-  res = res.body;
-  const response = JSON.parse(res);
+    res = res.body;
+    const response = JSON.parse(res);
```

Comment:
@rarkins We should refactor this to let got the json parsing by passing `json:true` as option